cli/build/register.ts

```diff
@@ -137,6 +139,13 @@ export const registerBuild = (program: Command) => {
                 filePath: normalizedSourcePath,
                 fileStaticAssetUrl: `./${normalizedOutputPath}`,
               })
+              if (buildOutcome.circuitJson) {
```

Comment:
incorrect name someone else introduced

```suggestion
              if (buildOutput.circuitJson) {
```